flink-libraries/flink-table/pom.xml

```diff
@@ -92,7 +92,12 @@ under the License.
 			</exclusions>
 		</dependency>
 
-
+		<dependency>
```

Comment:
fabian,about reflections jar, its license is WTFPL, WTFPL only says 'Everyone is permitted to copy and distribute verbatim or modified
copies of this license document', however it does not say it compatible with ASL explicitly.
Besides, I also want to include the class of this jar into the generated flink-table-{version}.jar, just like calcite class. I don’t know whether it is allowed.